src/Microsoft.MobileBlazorBindings.WebView/Elements/BlazorWebView.cs

```diff
@@ -298,6 +298,7 @@ private static Stream SupplyFrameworkFile(string uri)
             return uri switch
             {
                 "framework://blazor.desktop.js" => typeof(BlazorWebView<>).Assembly.GetManifestResourceStream("Microsoft.MobileBlazorBindings.WebView.blazor.desktop.js"),
+                "framework://blazor.desktop.js/" => typeof(BlazorWebView<>).Assembly.GetManifestResourceStream("Microsoft.MobileBlazorBindings.WebView.blazor.desktop.js"),
```

Comment:
Might be better to switch this back to a `switch statement` because a `switch expression` doesn't allow case fall-through.